SourceryTests/Stub/JavaScriptTemplates/Includes.js

```diff
@@ -0,0 +1 @@
+<%- include('Equality.js') %>-%><%- include('Other.js') %>-%>
```

Comment:
is it a valid syntax to have just a closing `-%>`? I though it needs open tag too, like `<% -%>`